src/pages/ProjectSettings.tsx

```diff
@@ -110,10 +112,18 @@ export const ProjectSettings: React.FC = () => {
     ? projectType === ProjectType.Project || repoData
     : repoData;
 
+  const repoDisplay = `${repoData?.repoSettings?.projectRef?.owner}/${repoData?.repoSettings?.projectRef?.repo}`;
+
   return (
     <ProjectSettingsProvider>
       <SideNav aria-label="Project Settings">
-        <SideNavGroup header="Project" />
+        <DropdownContainer>
+          <ProjectSelect
+            selectedProjectIdentifier={repoDisplay || identifier}
```

Comment:
`repoDisplay` will always be defined.  since be default it is "/". identifier will never be used.

Reply:
thank you for spotting that!